common/js/injected_script.js

```diff
@@ -432,6 +432,27 @@ class InjectedScript {
     node.dispatchEvent(event);
   }
 
+  setInputFiles(node, payloads) {
+    if (node.nodeType !== 1)
+      return "error:notelement";
```

Comment:
I just followed the pattern in this file. Every occurance is "error:notelement"

Reply:
@inancgumus wdyt is the best way forward with the errors in the injected script?

Reply:
Maybe I should rewrite the other two error to not contain spaces ('error:not input element' and 'error:Not an input[type=file] element')

Reply:
Don't make this change i've suggested? I've just found that we're relying on `error:notelement` [here](https://github.com/grafana/xk6-browser/blob/main/common/element_handle.go#L1547). Thanks!

Reply:
Thanks. Maybe I should then use error:notinput instead of error:notinputelement in line 439, and add error:notinputfileelement to the map in element_handle.go. @ankur22 what do you think?

Reply:
Yep, i like that idea 👍

Reply:
I have done this also